packages/react-img-viewer/src/ImageViewer.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useRef, useEffect } from 'react';
+import { FullScreenViewer, ImageViewer } from 'iv-viewer';
+import 'iv-viewer/dist/iv-viewer.css';
+import { ImageViewerProps } from './types';
+
+export const NormalViewer: React.FC<ImageViewerProps> = ({
```

Comment:
**Suggestion**

Let's keep the component name as `ImageViewer` only. As its causing conflict on import we can alias the imports.
```
```